### DISCUSSION TITLE
Monitoring in version 20

### DISCUSSION THREAD

**Discussion by saloni-2404** [strimzi/strimzi-kafka-operator#4138] [General]
From the blog I was able to see that monitoring using port 9404 has been deprecated in v20 of Kafka. What are the other ways in which we can proceed with monitoring v20 or is there any way to have 9404 along with v20?

**Comment by scholzj** [comment]
Not sure from which blog post you read it. The port 9404 is not exposed anymore on the services. But it is still exposed on the pods when you have metrics enabled. So you just have to configure your Prometheus to scrape the from the Pods directly. Please check the docs for the guidance how to do it with the Prometheus Operator and PodMonitors.

**Reply by saloni-2404** [reply]
ok Jakub, thanks, will check on that
